pylua/test/test_timer.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+
+import pylua.timer
+import logging
+import time
+import threading
+
+logger = logging.getLogger(__name__)
+
+class TestPyTimer:
+	__test_timer = None
+	__start_timer = None
+	__finish_timer = None
+	__interval = None
+	__cond_var = None
+
+	def timer_task(self):
+		self.__cond_var.acquire()
+		self.__finish_timer = float(time.time())
+		self.__cond_var.notify()
+		self.__cond_var.release()
+		pass
+
+	def __init__(self):
+		self.__interval = 1
+		self.__test_timer = timer.PyTimer(self.timer_task, self.__interval)
+		self.__cond_var = threading.Condition()
+		pass
+
+	def run_cond_var(self, wait_time):
+		self.__cond_var.acquire()
+		self.__cond_var.wait(wait_time)
+		self.__cond_var.release()
+		pass
+
+	def test_run_timer(self, interval = None):
```

Comment:
@dtrunov, if there will be single task for many test cases, then it's good to have some external private method for task. But in this case, after you'll rework tests, I think there will be no similarity. I think there will be diffident  tasks for each test case and in this case local function for task is better since in this case test case logic is localized. Plus you don't have to imagine unique name for each task, you can use `task` name for all.